lib/money-rails/mongoid/money.rb

```diff
@@ -48,23 +46,22 @@ def evolve(object)
     private
 
     def mongoize_hash(hash)
-      if hash.respond_to?(:deep_symbolize_keys!)
-        hash.deep_symbolize_keys!
-      elsif hash.respond_to?(:symbolize_keys!)
-        hash.symbolize_keys!
-      end
+      hash = hash.symbolize_keys
 
       # Guard for a blank form
       return nil if hash[:cents] == '' && hash[:currency_iso] == ''
 
-      ::Money.new(hash[:cents], hash[:currency_iso]).mongoize
+      # QUESTION: what if Money::Currency::UnknownCurrency will be raised?
+      # Should it be dependent on `MoneyRails.raise_error_on_money_parsing`?
```

Comment:
That's a great question. I think it makes sense not to raise when that option is on and return `nil`. I think it used to be the behaviour at some point, not sure why it changed. Might be worth digging through commit history

Reply:
Yeah, I guess it was me who actually broke it, will fix.